app/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import './style.css';
```

Comment:
Please, rename file extension to scss or remove the file. You app still doesn't handle css :)